osu.Game/Migrations/20171019041408_InitialCreate.Designer.cs

```diff
@@ -123,6 +123,8 @@ protected override void BuildTargetModel(ModelBuilder modelBuilder)
 
                     b.Property<string>("BackgroundFile");
 
+                    b.Property<string>("VideoFile");
```

Comment:
i don't think this needs to exist in the database

Reply:
it should? how can we know the video file of the beatmap